File: ExerciseNote.WebAppApi/Models/BodyDiary.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using System;

namespace ExerciseNote.WebAppApi.Models
{
    public class BodyDiary
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]// szöveges id generálás
        public string Id { get; set; }
        public string IdUser { get; set; }
        public int Age { get; set; }
        public int Height { get; set; }
        public int Weight { get; set; }
        public int BodyFat { get; set; }
      
        public int MaintainWeight { get; set; }
        public int WeightLoss { get; set; }
        public int WeightGain { get; set; }
        public DateTime Created_at { get; set; }
        public string PhotoId { get; set; }
        public Photo Photo { get; set; }

    }
}

File: ExerciseNote.WebAppUI/src/app/models/bodyDiary.ts
import { User } from "../shared/models/account/user";
import { Photo } from "./photo";

export class BodyDiary {
  public id: string;
  public idUser: string;
  public age: number;
  public height: number;
  public weight: number;
  public bodyFat: number;
  public photo: Photo;
  public maintainWeight: number ;
  public weightLoss: number ;
  public weightGain: number ;
  public created_at: string ;


  
}
